templates/openwisp2/settings.py

```diff
@@ -122,6 +123,10 @@
     },
 ]
 
+CELERY_TASK_ACKS_LATE = True
+CELERY_WORKER_PREFETCH_MULTIPLIER = 1
+CELERY_BROKER_URL = 'redis://127.0.0.1:6379/3'
```

Comment:
also make this configurable, variable name `openwisp2_celery_broker_url`, defaults to `redis://127.0.0.1:6379/3`.